script/foreman-debug

```diff
@@ -331,6 +331,7 @@ if [ $NOGENERIC -eq 0 ]; then
     [ $DEBUG -eq 0 ] && add_cmd "dpkg --list" "installed_packages"
   fi
 
+  add_cmd "virt-who -dop"
```

Comment:
No it's not. You're more likely to see it in Katello. However, it can be useful to know about the environment and AFAIK `add_cmd` gracefully fails if the command isn't present so I'd be ok with it.

Reply:
it's typically used when Katello is installed of foreman_virt_who_configure plugin is used (depends on Katello), :+1: from my side